<?xml version="1.0" encoding="UTF-8"?>

<launch>
	<!--IMU-->
	<!-- <node pkg="myahrs_driver" type="myahrs_driver" name="myahrs_driver">
		<param name="port" value="/dev/ttyIMU" />
		<param name="baud_rate" value="115200" />
	</node>
	<node pkg="tricat221"  type="heading_calculator.py" name="headingCalculator" respawn="true"/> -->

	<!--Visualization-->
	<!-- <node name="rqt" pkg="rqt_graph" type="rqt_graph"/> -->

	<!--Serovo Test-->
	<!-- <node pkg="tricat221" type="servo_test.py" name="servo_test" output="screen"/>
	<rosparam command="load" file="$(find tricat221)/params/servo_params.yaml"/>
	<node pkg="tricat221" type="serial_node.py" name="ARDUINO" args="/dev/ttyARDUINO" respawn="true"/> -->

	<!--LiDAR Test-->
	<node pkg="rosbag" type="play" name="rosbag_play" required="true" args="$(find tricat221)/etc/rosbag/0804_fountain_wrong_psi.bag -l --topics /scan"/>
	<node pkg="tricat221" type="lidar_converter.py" name="lidarConverter" respawn="true" output="screen"/>
	<rosparam command="load" file="$(find tricat221)/params/lidar_params.yaml"/>
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find tricat221)/rviz/rviz_setting1.rviz" /> -->

	<!--Cam Test-->
	<!-- <rosparam command="load" file="$(find tricat221)/params/docking_params.yaml"/>
	<rosparam command="load" file="$(find tricat221)/params/coordinates.yaml"/>
	<node pkg="rosbag" type="play" name="rosbag_play" required="true" args="$(find tricat221)/etc/rosbag/cam-rosbag/one_mark.bag -l--> <!--topics /usb_cam/image_raw"/>
	<node pkg="tricat221" type="camera_test.py" name="camera_test" output="screen" /> -->

</launch>
